python/paddle/v2/fluid/layers/nn.py

```diff
@@ -3091,3 +3094,41 @@ def multiplex(inputs, index):
                 'Ids': index},
         outputs={'Out': [out]})
     return out
+
+
+def flatten(input, axis=1):
+    """
+    **Flatten Layer**
+
+    Flatten is used to reshape the input. After flattening, input's shape will
+    be two-dimension. The axis is used to indicate where to split the dimension.
+
+    Args:
+       input(variable): The input tensor.
+       axis(int):
+
+    Returns:
+       Variable: A tensor variable.
+
+    Examples:
+       .. code-block:: python
+
+          x = fluid.layers.data(name="data", shape=[3, 32, 32], dtype="float32")
+          reshaped = fluid.layers.reshape_with_axis(input=x, axis=2)
+          reshaped.shape
+            >> [-1, 1024]
+    """
+    assert len(input.shape) > axis and axis > 0, \
+        "the axis should be litter than input.shape's."
```

Comment:
litter --> smaller
input.shape's --> arity of input's shape

Reply:
Done